src/queries/query-evaluation-model-in-detail.md

```diff
@@ -158,8 +158,9 @@ would still exist and already executed queries would not have to be re-done.
 
 ## Cycles
 
-Earlier we stated that query invocations form a DAG. However, it would be easy
-form a cyclic graph by, for example, having a query provider like the following:
+Earlier we stated that query invocations form a DAG. However, it would be easy 
+to form a cyclic graph by, for example, having a query provider like the 
```

Comment:
```suggestion
Earlier we stated that query invocations form a DAG. However, it would be easy
to form a cyclic graph by, for example, having a query provider like the
```

There're trailing spaces.